book/src/common_patterns/selection.md

```diff
@@ -62,8 +62,8 @@ This will result in a dependency graph like so:
 request_text  -->  parse  -->  header -->  content_type  -->  (other queries)
 ``` 
 
-The advantage of this is that changes that only effect the "body" of
-only consume small parts of itthe request will
+The advantage of this is that changes that only effect the "body" or
+only consume small parts of the request will
```

Comment:
Is this the correct meaning of this sentence? It seems to make sense to me but I'm not 100% certain.